spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -282,16 +284,15 @@ public void testCustomPartitionCountDoesNotOverridePartitioningIfSmaller() throw
 
 		byte[] ratherBigPayload = new byte[2048];
 		Arrays.fill(ratherBigPayload, (byte) 65);
-		KafkaTestBinder binder = (KafkaTestBinder) getBinder();
+		KafkaTestBinder binder = getBinder();
 
 		DirectChannel moduleOutputChannel = new DirectChannel();
 		QueueChannel moduleInputChannel = new QueueChannel();
-		Properties producerProperties = new Properties();
-		producerProperties.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "3");
-		producerProperties.put(BinderPropertyKeys.NEXT_MODULE_COUNT, "5");
-		producerProperties.put(BinderPropertyKeys.PARTITION_KEY_EXPRESSION, "payload");
-		Properties consumerProperties = new Properties();
-		consumerProperties.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "3");
+		KafkaProducerProperties producerProperties = new KafkaProducerProperties();
+		producerProperties.setPartitionCount(5);
```

Comment:
are we missing anything here by not having a new property equivalent for this?: `producerProperties.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "3");`